web_external/Datasets/DatasetsView.js

```diff
@@ -43,28 +43,27 @@ isic.views.DatasetsView = isic.View.extend({
 
         // Display loading indicator
         if (!this.loaded) {
-            new girder.views.LoadingAnimation({
+            this.loadingAnimation = new girder.views.LoadingAnimation({
                 el: this.$('.isic-listing-loading-animation-container'),
                 parentView: this
             }).render();
+        } else {
```

Comment:
I split this `else` / `if` on purpose, just because I think it's more understandable.